Development-Guide/AsciiDoc-Guide/README.adoc

```diff
@@ -77,7 +77,55 @@ For example:
 //Some c code
 ----
 
-== Formatting
+== Formatted Text
+
+=== Bold Text
+
+To bold text in an asciioc place a single asterisk on each side of the phrase or word to be bolded.
+
+----
+*Bolded Statement*
+----
+
+Will appear as:
+
+*Bolded Statemenet*
+
+Individual parts of a word can be bolded by breaking up with word with double asterisks.
```

Comment:
`breaking up with word with double asterisks` -> `breaking up the word with double asterisks`